easybuild/easyblocks/m/matlab.py

```diff
@@ -74,15 +74,16 @@ def configure_step(self):
         licport = self.cfg['license_server_port']
         if licport is None:
             licport = os.getenv('EB_MATLAB_LICENSE_SERVER_PORT', '00000')
```

Comment:
for consistency, everything regarding `licserv` and `licport` should also be inside `if licfile is None:` then, since it's only used there?